server/belga/io/feed_parsers/base_belga_newsml_1_2.py

```diff
@@ -74,9 +74,10 @@ def parse(self, xml, provider=None):
                     if not [it for it in item.get('subject', []) if it.get('scheme') == 'news_products']:
                         product = {"name": 'GENERAL', "qcode": 'GENERAL', "scheme": "news_products"}
                         item.setdefault('subject', []).append(product)
-                    # add service is NEW
-                    service = {"name": 'NEWS', "qcode": 'NEWS', "scheme": "news_services"}
-                    item.setdefault('subject', []).append(service)
+                    item['subject'].extend([
+                        {"name": 'default', "qcode": 'default', "scheme": "distribution"},  # Distribution is default
+                        {"name": 'NEWS', "qcode": 'NEWS', "scheme": "news_services"},  # add service is NEW
```

Comment:
`{"name": 'NEWS', "qcode": 'NEWS', "scheme": "news_services"}` is it for all parsers or only for DPA?

Reply:
Yes, it use for all for external source not Belga.